src/Spice86.Core/Emulator/Memory/XmsMoveData.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace Spice86.Core.Emulator.Memory;
+
+using System.Runtime.InteropServices;
+
+/// <summary>
+/// In-memory structure with information about an XMS move request.
+/// </summary>
+[StructLayout(LayoutKind.Sequential, Pack = 1)]
```

Comment:
To convert to emulated memory structure? See DosMemoryControlBlock for example

Reply:
Probably. It's also very close to what DOSBox does:

    #pragma pack (1)
    #endif
    struct XMS_MemMove{
    	uint32_t length;
    	uint16_t src_handle;
    	union {
    		RealPt realpt;
    		uint32_t offset;
    	} src;
    	uint16_t dest_handle;
    	union {
    		RealPt realpt;
    		uint32_t offset;
    	} dest;
    
    } GCC_ATTRIBUTE(packed);
    #ifdef _MSC_VER
    #pragma pack ()
    #endif